downstream/assemblies/platform/assembly-migrate-isolated-execution-nodes.adoc

```diff
@@ -0,0 +1,40 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-migrate-isolated-execution-nodes"]
+= Upgrading isolated nodes to execution nodes
+
+[role="_abstract"]
+When upgrading from version 1.x to the latest version of the {PlatformName}, platform administrators will need to migrate the data from their legacy isolated nodes into execution nodes, which will be necessary to deploy automation mesh.
```

Comment:
Suggest rewording for active voice:
Upgrading from version 1.x to the latest version of the {PlatformName} requires platform administrations to migrate data from isolated legacy nodes to execution nodes. This migration is necessary to deploy the automation mesh.